ZalikovkaC#/Program.cs

```diff
@@ -10,28 +10,30 @@ class Program
     {
         static void Main(string[] args)
         {    
-            int higerMath, physics, informatics, history, TCA, ecology, english, PE;
+            int higherMath, physics, informatics, history, TDA, ecology, english, PE;
```

Comment:
All of these are name of subjects. How to understand **by it's name** what information you want to keep in them? The number of students, number of room, where lessons are taken place?